hudai/resources/person_quote.py

```diff
@@ -6,29 +6,28 @@
 
 class PersonQuoteResource(Resource):
     def __init__(self, client):
-        Resource.__init__(self, client, base_path='/people/{person_id}/quotes')
+        Resource.__init__(self, client, base_path='/people/quotes')
         self.resource_name = 'PersonQuote'
 
     def list(self, person_id, article_id=None, term=None, page=None):
         query_params = self._set_limit_offset({'page': page})
+        query_params['person_id'] = person_id
         query_params['article_id'] = article_id
         query_params['term'] = term
```

Comment:
This can be simplified to

```python
query_params = self._set_limit_offset({
    'article_id': article_id,
    'person_id': person_id,
    'term': term,
    'page': page
})
```